src/installer/pkg/packaging/rpm/pack-deps.proj

```diff
@@ -0,0 +1,97 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <Import Project='package.props' />
+
+  <UsingTask TaskName="ReplaceFileContents" AssemblyFile="$(InstallerTasksAssemblyPath)"/>
+  <UsingTask TaskName="BuildFPMToolPreReqs" AssemblyFile="$(InstallerTasksAssemblyPath)"/>
```

Comment:
Missing space for style

```suggestion
  <UsingTask TaskName="BuildFPMToolPreReqs" AssemblyFile="$(InstallerTasksAssemblyPath)" />
```

Reply:
Sure - I can fix this - however it was a straight copy from https://github.com/dotnet/runtime/blob/master/src/installer/pkg/packaging/rpm/package.targets

Reply:
Removing this whole proj file.